plugins/lightspeed-backend/src/service/router.ts

```diff
@@ -0,0 +1,74 @@
+import { errorHandler } from '@backstage/backend-common';
+import { LoggerService } from '@backstage/backend-plugin-api';
+import { Config } from '@backstage/config';
+
+import express from 'express';
+import Router from 'express-promise-router';
+import { APIError, OpenAI } from 'openai';
+
+import { CompletionsRequestBody } from './types';
+import { validateCompletionsRequest } from './validation';
+
+export interface RouterOptions {
+  logger: LoggerService;
+  config: Config;
+}
+
+export async function createRouter(
+  options: RouterOptions,
+): Promise<express.Router> {
+  const { logger, config } = options;
+
+  const router = Router();
+  router.use(express.json());
+
+  router.get('/health', (_, response) => {
+    logger.info('PONG!');
+    response.json({ status: 'ok' });
+  });
+
+  router.post(
+    '/chat/completions',
+    validateCompletionsRequest,
```

Comment:
Maybe we can validate only via openai's `APIError`, or via using `openapi.yaml` schema and `openapi-backend`